js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired, AuthenticationError, DDoSProtection } = require ('./base/errors');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class bitopro extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'bitopro',
+            'name': 'BitoPro',
+            'countries': [ 'TW' ], // Taiwan
+            'version': 'v3',
+            'rateLimit': 600,
```

Comment:
I don't think this is correct:

Considering 600 req per minute (1000 milliseconds), we get 1 request every 1,6666 milliseconds (600/1000) 

![image](https://user-images.githubusercontent.com/43336371/154234045-1809c28d-63b3-4722-8525-65461fa56551.png)

Reply:
@sc0Vu `rateLimit` represents the value in milliseconds for a single request, so if we can fire 1 request every 1,6666 milliseconds that should be the value of the rateLimit